shared/utils/collections.ts

```diff
@@ -28,3 +28,16 @@ export const sortNavigationNodes = (
       : document.children,
   }));
 };
+
+export const colorPalette = () => [
```

Comment:
```suggestion
export const colorPalette = [
```

Just export an array, avoids needing the indirection on `Collection.COLOR_PALETTE` then

Reply:
Fixed!